dbos/_dbos.py

```diff
@@ -658,6 +715,25 @@ def set_event(cls, key: str, value: Any) -> None:
         """
         return set_event(_get_dbos_instance(), key, value)
 
+    @classmethod
+    async def set_event_async(cls, key: str, value: Any) -> None:
+        """
+        Set a workflow event.
+
+        This function is to be called from within a workflow.
+
+        `set_event` sets the `value` of `key` for the current workflow instance ID.
+        This `value` can then be retrieved by other functions, using `get_event` below.
+
+        Each workflow invocation should only call set_event once per `key`.
```

Comment:
Not actually true, you can call many times per key--if this was copied from regular `set_event`, let's fix it there.

Reply:
It was copy/pasted. Will fix